manimlib/mobject/svg/tex_mobject.py

```diff
@@ -134,13 +134,31 @@ def organize_submobjects_left_to_right(self):
 
 
 class TexMobject(SingleStringTexMobject):
+    """
+    `manimlib.mobject.svg.tex_mobject.TexMobject`
+    TexMobject
+    ==========
+    Writes to a LaTeX file in math mode
+    (equivalent to `\\textrm{}` or `$$` in LaTeX).\n
```

Comment:
Does the trailing newline need to be here, or in the other places in the docstring?

Reply:
Otherwise everything bunches up in one line when seen in as IDE.

Reply:
Something tells me we will have to come back to this trailing newlines when we move to sphinx

Reply:
^wdym "move", I mean we're already going to use it lol we have to see if this looks fine on sphinx later

Reply:
Which IDE are you using, and do other repositories do this? It seems wrong somehow.

Reply:
I'm using VSCode. Don't know about other libs.

Reply:
@PgBiel I guess I mean "have sphinx process this docstring". I agree with @eulertour this seems weird to me.